crates/boundless-market/src/contracts/mod.rs

```diff
@@ -786,235 +786,17 @@ pub fn eip712_domain(addr: Address, chain_id: u64) -> EIP721DomainSaltless {
     }
 }
 
+/// Constant to specify when no selector is specified.
+pub const UNSPECIFIED_SELECTOR: FixedBytes<4> = FixedBytes::<4>([0xFF; 4]);
```

Comment:
Now when no specific selector is specified, the value is 0xFFFFFFFF rather than all 0s. That is so that we can keep compatibility with the mock verifier and have more tools for our testing